README.yaml

```diff
@@ -126,9 +147,37 @@ usage: |-
   }
   ```
 
-# Example usage
-examples: |-
-  Review the [complete example](examples/complete) to see how to use this module.
+  ### Example of local maps configurations with importing and deepmerging
```

Comment:
I feel like without a YAML example, the explanation is lost.

For example, if you have a file like this (e.g. `myconfig.yaml`):
```yaml
import:
- file1
- file2
```

Then, this module will deep merge `file1.yaml` and `file2.yaml` into `myconfig.yaml`